config/fieldsToRedact.txt

```diff
@@ -1,38 +1,22 @@
-payload.data.coredata.angle
-payload.data.partII.value.lights.leftTurnSignalOn
-payload.data.partII.value.lights.rightTurnSignalOn
-payload.data.partII.value.lights.hazardSignalOn
-payload.data.partII.value.lights.fogLightOn
-payload.data.partII.value.lights.lowBeamHeadlightsOn
-payload.data.partII.value.lights.highBeamHeadlightsOn
-payload.data.partII.value.lights.automaticLightControlOn
-payload.data.partII.value.lights.daytimeRunningLightsOn
-payload.data.partII.value.lights.parkingLightsOn
-payload.data.partII.value.weatherProbe.rainRates.statusFront
-payload.data.partII.value.weatherProbe.rainRates.statusRear
-payload.data.partII.value.weatherProbe.rainRates.rateFront
-payload.data.partII.value.weatherProbe.rainRates.rateRear
-payload.data.partII.value.events.eventAirBagDeployment
-payload.data.partII.value.weatherReport.solarRadiation
-payload.data.partII.value.weatherReport.roadFriction
-payload.data.partII.value.weatherReport.friction
-payload.data.partII.value.weatherReport.airTemp
-payload.data.partII.value.weatherReport.airPressure
-payload.data.coreData.transmission
-payload.data.partII.value.speedProfile.speedReports
-payload.data.coreData.brakes.abs
-payload.data.partII.value.events.eventABSactivated
-payload.data.coreData.brakes.scs
-payload.data.partII.value.events.eventStabilityControlactivated
-payload.data.coreData.brakes.traction
-payload.data.partII.value.events.eventTractionControlLoss
-payload.data.coreData.brakes.brakeBoost
-payload.data.coreData.brakes.wheelBrakes.leftFront
-payload.data.coreData.brakes.wheelBrakes.rightFront
-payload.data.coreData.brakes.wheelBrakes.unavailable
-payload.data.coreData.brakes.wheelBrakes.leftRear
-payload.data.coreData.brakes.wheelBrakes.rightRear
-payload.data.coreData.brakes.auxBrakes
-payload.data.partII.value.events.eventFlatTire
-payload.data.partII.value.events.eventDisabledVehicle
-payload.data.partII.value.status.statusDetails
+payload.data.value.BasicSafetyMessage.coreData.angle
+payload.data.value.BasicSafetyMessage.coreData.transmission
+payload.data.value.BasicSafetyMessage.coreData.brakes.wheelBrakes
+payload.data.value.BasicSafetyMessage.coreData.brakes.traction
+payload.data.value.BasicSafetyMessage.coreData.brakes.abs
+payload.data.value.BasicSafetyMessage.coreData.brakes.scs
+payload.data.value.BasicSafetyMessage.coreData.brakes.brakeBoost
+payload.data.value.BasicSafetyMessage.coreData.brakes.auxBrakes
+payload.data.value.BasicSafetyMessage.partII.partII-Value.VehicleSafetyExtensions.events
+payload.data.value.BasicSafetyMessage.partII.partII-Value.VehicleSafetyExtensions.lights
```

Comment:
question: is this just redacting everything _under_ lights rather than iterating through all the values (presumably what we did before)?